### DISCUSSION TITLE
Where can I find descriptions of what the various models do

### DISCUSSION THREAD

**Discussion by RichCaloggero** [CarlGao4/Demucs-Gui#74] [Q&A]
I'm sure model names like "repro_mdx_a_hybrid_only" or "955717e8" mean something to someone, but meaningless to mere mortals like myself.

The extra info doesn't help much:

>Model signature: 955717e8
Type: Single model
Position: Local model
Repo: C:\Users\rich\Program Files\Demucs-GUI_1.2_cpu\pretrained
File: C:\Users\rich\Program Files\Demucs-GUI_1.2_cpu\pretrained\955717e8-8726e21a.th

Thanx for any info...

-- Rich

**Comment by CarlGao4** [comment]
These are model names defined by original demucs repo. models like `mdx_extra` `htdemucs_6s` are explained in official document, but `repro_mdx_a_hybrid_only` doesn't.
